navidrome/docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
```

I suggest using the app proxy instead of exposing the http port directly. This makes it more secure as the proxy is only for HTTP traffic and everything else can't get through
```suggestion
services:
  app_proxy:
    environment:
      APP_HOST: navidrome_navidrome_1
      APP_PORT: 4533
      PROXY_AUTH_ADD: "false"
```